.github/workflows/release-code-freeze.yml

```diff
@@ -21,142 +21,145 @@ env:
-        run: pnpm run version bump woocommerce -v ${{ needs.maybe-create-next-milestone-and-release-branch.outputs.next_version }}.0-dev
-
-      - name: Checkout pnpm-lock.yaml to prevent issues
-        run: git checkout pnpm-lock.yaml
-
-      - name: Commit changes
-        run: git commit -am "Prep trunk for ${{ needs.maybe-create-next-milestone-and-release-branch.outputs.next_version }} cycle"
-
-      - name: Push branch up
-        run: git push --no-verify origin prep/trunk-for-next-dev-cycle-${{ needs.maybe-create-next-milestone-and-release-branch.outputs.next_version }}
-
-      - name: Create the PR
-        uses: actions/github-script@v6
-        with:
-          script: |
-            const body = "This PR updates the versions in trunk to ${{ needs.maybe-create-next-milestone-and-release-branch.outputs.next_version }} for next development cycle."
-
-            const pr = await github.rest.pulls.create({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              title: "Prep trunk for ${{ needs.maybe-create-next-milestone-and-release-branch.outputs.next_version }} cycle",
-              head: "prep/trunk-for-next-dev-cycle-${{ needs.maybe-create-next-milestone-and-release-branch.outputs.next_version }}",
-              base: "trunk",
-              body: body
-            })
-
-  notify-slack:
-    name: "Sends code freeze notification to Slack"
-    if: ${{ inputs.skipSlackPing != true }}
-    runs-on: ubuntu-20.04
-    needs: maybe-create-next-milestone-and-release-branch
-    steps:
-      - name: Slack
-        uses: archive/github-actions-slack@v2.0.0
-        id: notify
-        with:
-          slack-bot-user-oauth-access-token: ${{ secrets.CODE_FREEZE_BOT_TOKEN }}
-          slack-channel: ${{ inputs.slackChannelOverride || secrets.WOO_RELEASE_SLACK_CHANNEL }}
-          slack-text: |
-            :warning-8c: ${{ needs.maybe-create-next-milestone-and-release-branch.outputs.release_version }} Code Freeze :ice_cube:
-            
-            The automation to cut the release branch for ${{ needs.maybe-create-next-milestone-and-release-branch.outputs.release_version }} has run. Any PRs that were not already merged will be a part of ${{ needs.maybe-create-next-milestone-and-release-branch.outputs.next_version }} by default. If you have something that needs to make ${{ needs.maybe-create-next-milestone-and-release-branch.outputs.release_version }} that hasn't yet been merged, please see the <${{ secrets.FG_LINK }}/code-freeze-for-woocommerce-core-release/|fieldguide page for the code freeze>.
-
-  trigger-changelog-action:
-    name: "Trigger changelog action"
-    runs-on: ubuntu-20.04
-    needs: maybe-create-next-milestone-and-release-branch
-    steps:
-      - run: |
-          curl \
-            -X POST \
-            -H "Accept: application/vnd.github.v3+json" \
-            -H "Authorization: token ${{ secrets.WC_BOT_TRIAGE_TOKEN }}" \
```

Comment:
Its not obvious in the changeset due to prettier reformatting files, but this is the token we're trying to get rid of.